packages/flame_3d/example/lib/rotating_light.dart

```diff
@@ -0,0 +1,20 @@
+import 'dart:math';
+
+import 'package:flame_3d/components.dart';
+import 'package:flame_3d/game.dart';
+
+class RotatingLight extends LightComponent {
+  RotatingLight()
+      : super.spot(
+          position: Vector3.zero(),
+        );
+
+  @override
+  void update(double dt) {
+    const radius = 15;
+    final angle = DateTime.now().millisecondsSinceEpoch / 4000;
+    final x = cos(angle) * radius;
+    final z = sin(angle) * radius;
+    position.setValues(x, 10, z);
```

Comment:
A little TODO to say that this is temporary would be nice :P the CI fails already anyway

Reply:
wouldn't say it is temporary, it is just an example

is the CI failing for this? I can only see the "expected" failures